torch/csrc/jit/fuser/arg_spec.h

```diff
@@ -18,9 +18,16 @@ namespace fuser {
 // Note: the device to run on is included in the arg spec because kernels
 //  are compiled per-device.
 struct TORCH_API ArgSpec {
-  ArgSpec(at::TensorList inputs, const int _device)
+  ArgSpec(
+      at::TensorList inputs,
+      const int _device,
+      const std::vector<int64_t> list_input_lengths)
```

Comment:
You can't just discard those, because they are needed in `operator==` in case of hash collisions.

Reply:
Also, you make a copy unnecessarily. Probably wanted to take a reference